cinch/roles/jenkins_master/handlers/main.yml

```diff
@@ -11,7 +11,7 @@
     name: jenkins
     state: restarted
 
-- name: save iptables
-  command: service iptables save
+- name: reload firewalld
```

Comment:
I know that we haven't done this much in the past, but any time we can't use an Ansible built-in and/or have to tag something with skip_ansible_lint, we should probably add a comment to explain why decisions were made for future maintainability.